IreneBranch/Project1

```diff
@@ -0,0 +1 @@
+Subproject commit b9f80c09e9ca3f8f2bb55e9b79e78a0e2bcb6996
```

Comment:
When I do git merge newBranch from master it says already up to date, but
the files are slightly different (one blank space at the beginning of the
last paragraph. It looks like I did change something in master but not sure
how or when

When I go to master or do git status it says
Your branch and 'origin/master' have diverged,
and have 2 and 1 different commit each, respectively.
  (use "git pull" to merge the remote branch into yours)

But when I do push or merge it says everything is fine. What can I do?

On 2 May 2016 at 19:46, Ariadni-Karolina Alexiou <notifications@github.com>
wrote:

> In IreneBranch/Project1
> <https://github.com/IreneG5/Project1/pull/2#discussion_r61783294>:
>
> > @@ -0,0 +1 @@
> > +Subproject commit b9f80c09e9ca3f8f2bb55e9b79e78a0e2bcb6996
>
> ok, well done! but now the branch has merge conflicts. Not sure why
> though. Did you update the master branch after your branched off to this
> branch? Try to use the command line to resolve the issue, as github
> suggests.
> … <#m_-3908098943051204967_>
> On Mon, May 2, 2016 at 9:33 PM, IreneG5 ***@***.***> wrote: In
> IreneBranch/Project1 <#2 (comment)
> <https://github.com/IreneG5/Project1/pull/2#discussion_r61781311>>: > @@
> -0,0 +1 @@ > +Subproject commit b9f80c0
> <https://github.com/IreneG5/Project1/commit/b9f80c09e9ca3f8f2bb55e9b79e78a0e2bcb6996>
> done! (I think) … <#m_-8547804738445428477_> On 2 May 2016 at 19:22,
> Ariadni-Karolina Alexiou ***@***.***> wrote: In IreneBranch/Project1 <#2
> <https://github.com/IreneG5/Project1/pull/2> (comment) <#2 (comment)
> <https://github.com/IreneG5/Project1/pull/2#discussion_r61779610>>>: > @@
> -0,0 +1 @@ > +Subproject commit b9f80c0
> <https://github.com/IreneG5/Project1/commit/b9f80c09e9ca3f8f2bb55e9b79e78a0e2bcb6996>
> <b9f80c0
> <https://github.com/IreneG5/Project1/commit/b9f80c09e9ca3f8f2bb55e9b79e78a0e2bcb6996>>
> git rm this folder from your branch — You are receiving this because you
> authored the thread. Reply to this email directly or view it on GitHub <
> https://github.com/IreneG5/Project1/pull/2/files/ea58a2dc7f0a59d8ae8da51f7d79e2a432cab3a1#r61779610>
> — You are receiving this because you were mentioned. Reply to this email
> directly or view it on GitHub <
> https://github.com/IreneG5/Project1/pull/2/files/ea58a2dc7f0a59d8ae8da51f7d79e2a432cab3a1#r61781311>
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/IreneG5/Project1/pull/2/files/ea58a2dc7f0a59d8ae8da51f7d79e2a432cab3a1#r61783294>
>